image_client/db_utils.py

```diff
@@ -45,41 +48,39 @@ def reset_connection(self):
         self.connect()
 
     def connect(self):
-
         if self.cnx is None:
             self.logger.debug(f"Connecting to db {self.database_host}...")
 
             try:
-                self.cnx = mysql.connector.connect(user=self.database_user,
-                                                   password=self.database_password,
-                                                   host=self.database_host,
-                                                   port=self.database_port,
-                                                   database=self.database_name)
-            except mysql.connector.Error as err:
-                if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+                self.cnx = mariadb.connect(user=self.database_user,
+                                           password=self.database_password,
+                                           host=self.database_host,
+                                           port=self.database_port,
+                                           database=self.database_name)
+            except mariadb.Error as err:
+                print("oh no! error")
+                print(err)
+                if err.errno == 1045:
                     self.logger.error(f"Starting client...")
-
                     self.logger.error("SQL: Access denied")
-                elif err.errno == errorcode.ER_BAD_DB_ERROR:
+                elif err.errno == 1049:
                     self.logger.error("Database does not exist")
                 else:
                     self.logger.error(err)
                 sys.exit(1)
-            except Exception as err:
-                self.logger.error(f"Unknown exception: {err}")
-                sys.exit(1)
 
             self.logger.info("Db connected")
         else:
             pass
             # self.logger.debug(f"Already connected db {self.database_host}...")
 
-
-
-
+    # delarocamcas changed to buffered = True, so will work inside forloops
     def get_one_record(self, sql):
```

Comment:
This is the same as "execute" below. Put error catch in that funciton and remove this one, update references.